specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-10-01-preview/containerInstance.json

```diff
@@ -0,0 +1,2605 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/NetworkDependenciesResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/subnets/{subnetName}/providers/Microsoft.ContainerInstance/serviceAssociationLinks/default": {
+      "delete": {
+        "operationId": "SubnetServiceAssociationLink_Delete",
+        "x-ms-examples": {
+          "SubnetServiceAssociationLinkDelete": {
+            "$ref": "./examples/SubnetServiceAssociationLinkDelete.json"
+          }
+        },
+        "summary": "Delete container group virtual network association links.",
+        "description": "Delete container group virtual network association links. The operation does not delete other resources provided by the user.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/VirtualNetworkNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubnetNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Delete started."
+          },
+          "204": {
+            "description": "No Content - the specified container group was not found."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "SecretReference": {
+      "description": "A secret reference",
+      "type": "object",
+      "required": [
+        "name",
+        "identity",
+        "secretReferenceUri"
+      ],
+      "properties": {
+        "name": {
+          "description": "The identifier of the secret reference",
+          "type": "string"
+        },
+        "identity": {
```

Comment:
>identity [](http://example.com/codeflow?start=9&length=8)

is this of arm-id format?

Reply:
yes here are examples https://github.com/Azure/azure-rest-api-specs/pull/30569/files/f40381f8cfd1691cde026e4c6a4ee22335d8ae46#diff-0ab5ad790563c6d3a8cf0c2af176693c297d197aa3c5165f6d75e07a8a6ccf1b:~:text=%22secretReferenceUri%22%3A%20%22https%3A//keyvaultname.vault.azure.net/secrets/envSecret%22

Reply:
I believe Razvan was asking about changing the format to "arm-id".

Reply:
resolved